docs/UserGuide.md

```diff
@@ -8,34 +8,33 @@
 2. [Quick Start](#2-quick-start)
 3. [Features](#3-features)
 <br/>&nbsp;3.1 [View the help: `help`](#31-view-the-help-help)
-<br/>&nbsp;3.2 [Adding a user: `addws`](#32-adding-a-user-adduser)
-<br/>&nbsp;3.3 [Switching users: `switchws`](#33-switching-users-switchuser)
-<br/>&nbsp;3.4 [Browse through all Anime: `browse`](#34-browse-through-all-anime-browse)
-<br/>&nbsp;3.5 [Watchlist management: `watchlist`](#35-watchlist-management-watchlist)
-<br/>&nbsp;3.6 [Add an Anime to the current watchlist: `add`](#36-add-an-anime-to-the-current-watchlist-add)
-<br/>&nbsp;3.7 [Remove an Anime from the current watchlist: `remove`](#37-remove-an-anime-to-the-current-watchlist-remove)
-<br/>&nbsp;3.8 [View all anime in watchlist: `view`](#38-view-all-anime-in-watchlist-view)
-<br/>&nbsp;3.9 [Bookmark an Anime: `bookmark`](#39-bookmark-an-anime-bookmark)
-<br/>&nbsp;3.10 [Search: `search`](#310-search-search)
-<br/>&nbsp;3.11 [View the information of an Anime: `info`](#311-view-the-information-of-an-anime-info)
-<br/>&nbsp;3.12 [Estimate time need to translate script: `estimate`](#312-estimate-time-needed-to-translate-script-estimate)
-<br/>&nbsp;3.13 [Exit AniChan: `exit`](#313-exit-anichan-exit)
-<br/>&nbsp;3.14 [Saving and loading data](#314-saving-and-loading-data)
+<br/>&nbsp;3.2 [Workspace management: `workspace`](#32-workspace-management-workspace)
+<br/>&nbsp;3.4 [Browse through all Anime: `browse`](#33-browse-through-all-anime-browse)
+<br/>&nbsp;3.5 [Watchlist management: `watchlist`](#34-watchlist-management-watchlist)
```

Comment:
It seems that there is a mistake in the numbering here

Reply:
You're right, shall leave the fix for the next update!